mentat/conversation.py

```diff
@@ -104,16 +145,16 @@ def get_model_response(self, config: ConfigManager) -> tuple[str, list[CodeChang
 
         messages.append({"role": "system", "content": system_message})
 
-        model, num_prompt_tokens = choose_model(messages, self.allow_32k)
+        num_prompt_tokens = get_prompt_token_count(messages, self.model)
 
         state = run_async_stream_and_parse_llm_response(
-            messages, model, self.code_file_manager
+            messages, self.model, self.code_file_manager
         )
 
         self.cost_tracker.display_api_call_stats(
             num_prompt_tokens,
-            count_tokens(state.message),
-            model,
+            count_tokens(state.message, self.model),
```

Comment:
I could be wrong, but I think using any model other than GPT 3.5 or 4 will crash here; inside the cost tracker, we just access a dictionary with the model as the key. Since we don't know how much it costs for models other than GPT, I think we should disable the cost tracker altogether for other models.

Reply:
Wait, somehow I didn't notice you changed the CostTracker! Sorry, you can ignore this!